server/services/participants-helper.js

```diff
@@ -56,6 +56,34 @@ const addDistanceToParticipantFields = (raw, siteDistanceJoin) =>
     }),
   }));
 
+// Find the previous status for each org, create a copy of it 
+const  revertToOldStatus = async ( participantStatuses,setParticipantStatus)=>{
```

Comment:
I think this is actually a change in the existing behaviour, but that's fine as long as we document it (I think before the participant would always go back to "prospecting" - my understanding here is they would go back to any prior status at all, although I believe the button is disabled for withdrawn archived participants so going back to "hired" would never be possible)

Reply:
It's pretty clear this was a case that wasn't thought of when it was first implemented. 

The previous function was to set the status of the `participant` to withdrawn but not update any `participants_status`, hence the bug.  I added this part to ensure that the ability to re-engage a user after they're withdrawn by a MoH user was kept.

This all might warrant further discussion.

Reply:
Discussed offline - MOH undoing a withdrawal is not a real use case. For now we are documenting this behaviour but it should be disabled in future.